requirements.txt

```diff
@@ -2,3 +2,6 @@ hwi
 flask
 python-bitcoinrpc
 ckcc-protocol
+flask_json_schema
+# with simplejson flask can serialize decimals
+simplejson
```

Comment:
Versions should probably be pegged in here.

Reply:
agreed